src/vos/tests/vos_size_dfs_sample.c

```diff
@@ -0,0 +1,297 @@
+/**
+ * (C) Copyright 2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+/* This generates a summary of struct sizes to be used by vos_estimate.py
+ * to generate metadata overhead estimates
+ */
+#define D_LOGFAC	DD_FAC(tests)
+
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include <errno.h>
+#include <getopt.h>
+#include <daos/debug.h>
+#include <daos.h>
+#include <daos_fs.h>
+
+#define DEFAULT_DFS_EXAMPLE_NAME "vos_dfs_sample.yaml"
+
+
+int
```

Comment:
(style)  externs should be avoided in .c files

Reply:
Function declaration moved to an internal header